assets/in

```diff
@@ -100,6 +100,9 @@ jq -n "{
     digest: $(echo $digest | jq -R .)
   },
   metadata: [
+    { name: \"repository\", value: \"${repository}\" },
+    { name: \"digest\", value: \"${digest}\" },
+    { name: \"tag\", value: \"${tag}\" },
```

Comment:
```suggestion
    { name: \"repository\", value: $(echo $repository | jq -R .) },
    { name: \"tag\", value: $(echo $tag | jq -R .) },
```
`digest` is already showing in the version so I think we don't need to include it in metadata again.

Is there an easy way to omit `image` when its value is null if `skip_download` is true?

Reply:
> digest is already showing in the version so I think we don't need to include it in metadata again.

if it can be used in the next step easily, then ok.

Reply:
Another reason behind is concourse/registry-image-resource always show `repository` and `tag` both in `in` and `out` without `image_id` and `digest` regardlessly. Since these two resource types are basically serving the same purpose I feel unifying the interface would help user to migrate from docker-image-resource to registry-image-resource.